src/main/scala/cromwell/engine/workflow/ExecutionStoreKey.scala

```diff
@@ -0,0 +1,66 @@
+package cromwell.engine.workflow
+
+import cromwell.binding._
+import cromwell.engine.ExecutionStatus
+import cromwell.engine.workflow.WorkflowActor.ExecutionStore
+
+sealed trait ExecutionStoreKey {
+  val scope: Scope
+
+  val index: Option[Int]
+
+  val parent: Option[ExecutionStoreKey]
```

Comment:
would there be strenuous objections to me removing the parent on my branch to stick with single-level scatter for now?  :smile: